libs/cardano-ledger-api/test/Test/Cardano/Ledger/Api/State/QuerySpec.hs

```diff
@@ -47,175 +47,203 @@ spec = do
           `shouldBe` getFilteredDelegationsAndRewardAccounts umap creds
 
   describe "GetCommitteeMembersState" $ do
-    committeeMembersStateSpec
+    committeeMembersStateSpec @Conway
 
-committeeMembersStateSpec :: Spec
+committeeMembersStateSpec ::
+  forall era.
+  ( EraTxOut era
+  , EraGov era
+  , Arbitrary (PParams era)
+  , Arbitrary (TxOut era)
+  , Arbitrary (Value era)
+  , Arbitrary (GovState era)
+  , Arbitrary (StashedAVVMAddresses era)
+  , Show (StashedAVVMAddresses era)
+  ) =>
+  Spec
 committeeMembersStateSpec =
-  prop "CommitteeMembersState Query" $ do
-    forAll (arbitrary @(NewEpochState Conway)) $ \nes' -> do
-      forAll
-        ( arbitrary
-            @( Set (Credential 'ColdCommitteeRole StandardCrypto)
-             , Set (Credential 'HotCommitteeRole StandardCrypto)
-             , Set MemberStatus
-             )
-        )
-        $ \(ckFilter', hkFilter', statusFilter) -> do
-          -- replace some arbitrary number of cold keys from the committeeState with the ones from the committee
-          -- so we can have Active members
-          let (comMembers, comStateMembers) = committeeInfo nes'
-          forAll (genCommonMembers comMembers comStateMembers) $ \newCommitteeState -> do
-            let nes =
-                  nes'
-                    & nesEpochStateL . esLStateL . lsCertStateL . certVStateL . vsCommitteeStateL
-                      .~ newCommitteeState
-            -- replace some cold and hot keys from the filter with known ones from both committee and committeeState
-            forAll
-              ( (,)
-                  <$> genRelevantColdCredsFilter ckFilter' comMembers comStateMembers
-                  <*> genRelevantHotCredsFilter hkFilter' comStateMembers
-              )
-              $ \(ckFilter, hkFilter) -> do
-                propEmpty nes
-                propComplete nes
-                propAuthorized nes
-                propActiveAuthorized nes
-                propNotAuthorized nes
-                propResigned nes
-                propUnrecognized nes
-                propFilters ckFilter hkFilter statusFilter nes
+  prop "CommitteeMembersState Query" $ \nes' ckFilter' hkFilter' statusFilter -> do
+    -- replace some arbitrary number of cold keys from the committeeState with the
+    -- ones from the committee so we can have Active members
```

Comment:
Maybe these comments make more sense a  few lines below, before[ this line ](https://github.com/input-output-hk/cardano-ledger/pull/3808/commits/926a9ec793d3e2b2d80d95b1e29dc23e2fe8689c#diff-fb722545a2358df077b89f77d538341ddfbb9d6a3e68354459abad096b1bfbc7R71) that does the replacing?

Reply:
Done